tests/command_count_test.go

```diff
@@ -0,0 +1,34 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCount(t *testing.T) {
+	connection := getLocalConnection()
+	responseValue := fireCommand(connection, "COMMAND COUNT")
+	responseCount  := responseValue.(int64)
+	if responseValue == nil {
+		t.Fail()
+	}
+
+	assert.Assert(t, responseCount > 0,
+		fmt.Sprintf("Unexpected number of CLI commands found. expected greater than 0, %d found", responseCount))
+}
+
+func getCommandCount(t *testing.B) {
```

Comment:
This function can probably be reused for the TestCommandCount method as well.

Reply:
Removed duplicate code and pushed new changes.